### DISCUSSION TITLE
Missing treads of agent runs

### DISCUSSION THREAD

**Discussion by prajwal-svl** [langchain-ai/langsmith-sdk#1078] [Q&A]
I recently started using Langgraph cloud for deploying my agents. I am only able to see the threads run from the last 14 days, the recent threads are not visible. Although the I have the thread ids, the UI is empty. What is the reason recent runs are not visible?

The threads visible:
![Screenshot 2024-10-08 at 11 57 08 AM](https://github.com/user-attachments/assets/793b396b-893e-4747-b91d-3718f0982450)

Missing Thread ID, although I see it in logs:
![Screenshot 2024-10-08 at 11 59 20 AM](https://github.com/user-attachments/assets/904f2c85-792d-47d7-8b8a-9906252f6221)

![Screenshot 2024-10-08 at 12 03 09 PM](https://github.com/user-attachments/assets/1f96a2a6-f114-4de9-a81f-03d4e0a7c210)

Also the monitor graph is empty:
![Screenshot 2024-10-08 at 1 17 25 PM](https://github.com/user-attachments/assets/1fe8659c-1c1b-4341-a8d9-beac5c581f8d)

Note: If this is not the right place for this Question. Please redirect to right one

**Comment by hinthornw** [comment]
Default data retention is 14 days:
https://docs.smith.langchain.com/administration/tutorials/manage_spend#keep-around-a-percentage-of-traces-for-extended-data-retention
